scipy/stats/tests/test_continuous.py

```diff
@@ -1148,6 +1148,80 @@ def pdf(self, x, a, b):
                 assert_allclose(X.moment(order, kind=kind),
                                 Y.moment(order, kind=kind))
 
+    # pdf and cdf formulas below can warn on boundary of support in some cases.
+    # See https://github.com/scipy/scipy/pull/22560#discussion_r1962763840.
+    @pytest.mark.filterwarnings("ignore::RuntimeWarning")
+    @pytest.mark.parametrize("c", [-1, 0, 1, np.asarray([-2.1, -1., 0., 1., 2.1])])
+    def test_custom_variable_support(self, c):
```

Comment:
Does this run reasonably fast locally?

Reply:
```
python dev.py test -t scipy.stats.tests.test_continuous -- -k custom_variable_support
```

runs in 0.67 seconds locally for me. I guess that may be a little too slow?